.github/ISSUE_TEMPLATE/other.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Other questions
+about: Governance of core libraries other than `base`; CLC policies and administration.
+title: ''
+labels: ''
+assignees: ''
+---
+
+<!--
+Thanks for raising an issue.
+
+Before proceeding, please make sure that you read [`README.md`](https://github.com/haskell/core-libraries-committee#readme) and [`PROPOSALS.md`](https://github.com/haskell/core-libraries-committee/blob/main/PROPOSALS.md).
```

Comment:
```suggestion
Before proceeding, please make sure that you read and understand [`README.md`](https://github.com/haskell/core-libraries-committee#readme) and [`PROPOSALS.md`](https://github.com/haskell/core-libraries-committee/blob/main/PROPOSALS.md).
```

Reply:
Not quite: it's fair to ask a question if you read but do not understand ;)